R/package.R

```diff
@@ -15,6 +15,9 @@
 #' use_package("dplyr", "suggests")
 #' }
 use_package <- function(package, type = "Imports") {
+  refuse_package(package, verboten = "tidyverse")
+  refuse_package(package, verboten = "devtools")
```

Comment:
Is it too early to do this for devtools, i.e. before more of the pieces have made it to CRAN?

Reply:
Yeah, I think it's too early.